pennylane/__init__.py

```diff
@@ -133,6 +136,97 @@ def device(name, *args, **kwargs):
     )
 
 
+def load(quantum_circuit_object, format: str):
+    """load(quantum_circuit_object, format)
+    Load external quantum assembly and quantum circuits from supported frameworks
+    into PennyLane templates.
+
+    .. note::
+    
+        For more details on which conversions are supported
+        please consult the corresponding plugin documentation:
+        https://pennylane.ai/plugins.html
+
+    **Example:**
+
+    >>> qc = qiskit.QuantumCircuit(2)
+    >>> qc.rz(0.543, [0])
+    >>> qc.cx(0, 1)
+    >>> my_circuit = qml.load(qc, format='qiskit')
+    >>>
+    >>> @qml.qnode(dev)
+    >>> def circuit(x):
+    >>>     qml.RX(x, wires=1)
+    >>>     my_circuit(wires=(1, 0))
+    >>>     return qml.expval(qml.PauliZ(0))
+
+    Args:
+        quantum_circuit_object: the quantum circuit that will be converted
```

Comment:
I guess we don't want to be to specific about type here?

Reply:
I would say that that is indeed the "easy" way out. Otherwise, the docstring would need to be updated each time in PL core once a new plugin supports conversion